index.bs

```diff
@@ -5294,6 +5295,13 @@ partial interface MLGraphBuilder {
     1. If [=MLGraphBuilder/validating operand=] with [=this=] and |input| returns false, then [=exception/throw=] a {{TypeError}}.
     1. If any of |sizes|'s [=list/items=] are 0, then [=exception/throw=] a {{TypeError}}.
     1. If |starts|'s [=list/size=] and |sizes|'s [=list/size=] are not both equal to |input|'s [=MLOperand/rank=], then [=exception/throw=] a {{TypeError}}.
+    1. [=list/For each=] |index| in [=the range=] 0 to |input|'s [=MLOperand/rank=], exclusive:
+        1. If |sizes|[|index|] is 0, then [=exception/throw=] a {{TypeError}}.
+
+            Issue(391): If 0-size dimensions are allowed, revise these steps.
+
+        1. If |starts|[|index|] is greater than or equal to |input|'s [=MLOperand/shape=][|index|], then [=exception/throw=] a {{TypeError}}.
```

Comment:
Is this one technically redundant because the line below will validate it?
Otherwise if we keep this line, then the same issue 391 applies to it too, as it starts[index] == shape[index] is okay given a 0 size.

Reply:
Shrug, I wager when we update line 5299, we'll notice this line too. So, let's just merge it now.

Reply:
Agreed that it's technically redundant, but I thought we'd revise everything after 391 and in that case we would want to consider the validation carefully.  Thanks for the merge!